nautobot/dcim/api/serializers.py

```diff
@@ -199,6 +199,18 @@ class Meta:
         list_display_fields = ["name", "status", "parent", "tenant", "description", "tags"]
         # https://www.django-rest-framework.org/api-guide/validators/#optional-fields
         validators = []
+        detail_view_config = [
+            {
+                "Location": {"fields": ["name", "display", "rack_count", "parent"]},
+                "Management": {"fields": ["status", "prefix_count", "device_count"]},
+                "Others": {"fields": ["tenant", "natural_key_slug", "url"]},
```

Comment:
We should think about whether we can have a pattern for "standard" fields to not need to be explicitly called out in the detail_view_config (i.e. `natural_key_slug`, `url`, `display`, `status` should always be handled consistently, so it's redundant and error-prone to have to call them out in every serializer).

Reply:
Excellent idea, but where would these fields be located?
I'm presuming it's typically on Col 1 Group 1?

Reply:
It's going to depend on the field:

- Status and Display appear in the page header already and probably can be simply ignored in the detail_view_config
- ~natural key slug~ ("composite key") and URL and ID probably should all appear at the end of column 1 group 1.